README.md

```diff
@@ -115,15 +367,21 @@ You can do this by going through these steps:
 - Make a new PR with the change
 - Have the PR get reviewed and merged
 
-### Known installation issues
+## Known Installation Issues
```

Comment:
Should we add a note that PyCharm users have had some issues for unknown reasons?

Reply:
Yes, thanks for flagging that!

Reply:
Additional info has been added.